packages/components/nodes/chains/SqlDatabaseChain/SqlDatabaseChain.ts

```diff
@@ -145,13 +219,17 @@ const getSQLDBChain = async (databaseType: DatabaseType, url: string, llm: BaseL
     )
 
     const db = await SqlDatabase.fromDataSourceParams({
-        appDataSource: datasource
+        appDataSource: datasource,
+        includesTables: includesTables,
+        ignoreTables: ignoreTables,
+        sampleRowsInTableInfo: sampleRowsInTableInfo
```

Comment:
have you tested if `sampleRowsInTableInfo` is working? I think we might have to do `parseInt(sampleRowsInTableInfo, 10)`

Reply:
sampleRowsInTableInfo and topK are already a numeric value